README.md

```diff
@@ -56,6 +56,20 @@ All html patterns in your pattern library will be located in `./patterns`. They
 
 Any changes to files in the `./patterns` directory will automatically be imported into your local Pattern Lab instance as long as you have run the `gulp serve` command and your server is running.
 
+## Can I include one pattern in another?
+
+as a twig include, yes.
+
+ * [how to twig includes](http://twig.sensiolabs.org/doc/tags/include.html)
+ * [example pattern with include](https://github.com/pattern-library/pattern-library/blob/master/patterns/molecules/media/figure-image/figure-image.twig)
+
+## Can I modify an existing pattern?
```

Comment:
Correct. All the patterns on a new install are coming from the github base pattern library repo that you worked on forever ago:

https://github.com/pattern-library/pattern-library

Specifically, the promo is here:
https://github.com/pattern-library/pattern-library/tree/master/patterns/molecules/components/promo

When you do an install, the PLibrary system looks for all patterns in node_modules and in ./patterns and injects them into Pattern Lab. So, on a brand-new fresh install from the Yeoman Generator, the *only* patterns that show up in Pattern Lab are from the original, node_modules-based pattern-library.

Over the next week I'll be working on a new sub-generator to clone *from* a node_modules-based pattern-library *into* the local `./patterns` folder.